tests/irepela/homework_4/test_sequences.py

```diff
@@ -11,7 +11,8 @@
     (2, 5, 1, test_list, [3, 4, 5]),
 ])
 def test_sequence_slice(start, end, step, collection, expected):
-    assert collection[start:end:step] == expected
+    test_slice = slice(start, end, step)
+    assert collection[test_slice] == expected

```

Comment:
using slice you could reduce amount of parameters in paramatrize, but its also fine

Reply:
try to group input parameters if they are bound logically